gensrc/thrift/InternalService.thrift

```diff
@@ -327,6 +327,8 @@ struct TQueryOptions {
 struct TScanRangeParams {
   1: required PlanNodes.TScanRange scan_range
   2: optional i32 volume_id = -1
+  3: optional bool placeholder = false
```

Comment:
what's this?

Reply:
It means: this scan range is just a placeholder without any scan range data in it.

it's set at the end of scan range list:  which means if this scan node has more scan ranges.

Reply:
I don't think the placeholder is good name, how about is_empty ?

Reply:
No problem, I can fix that.